rust/src/dns/dns.rs

```diff
@@ -381,7 +381,7 @@ impl DNSState {
         None
     }
 
-    fn parse_request(&mut self, input: &[u8], is_tcp: bool) -> bool {
+    fn parse_request(&mut self, input: &[u8], is_tcp: bool, fid: Option<Frame>, flow: *const core::Flow,) -> bool {
```

Comment:
This is even more nit than the other comment, and not a blocker. Why `fid` and not `frame`?

if this isn't something worth changing, all looks good to me :)

Reply:
Because I thought first I would only use the frame id...
Changing that